readthedocs/projects/tasks/builds.py

```diff
@@ -520,7 +522,10 @@ def on_failure(self, exc, task_id, args, kwargs, einfo):
             )
 
         # Update build object
-        self.data.build['success'] = False
+        if isinstance(exc, BuildUserSkip):
+            self.data.build["success"] = True
+        else:
+            self.data.build["success"] = False
```

Comment:
This should be moved to line 510 where we are already checking for this `isinstance`, using the same pattern we are following for `status` there.

Reply:
Some times your comments go for consistency and to not change existing patterns. But here, it seems to be the opposite :wink: 

I will revert this part of the change, perhaps the intention is to have `            self.data.build["success"] = False` no matter what. And this wasn't part of the initial bug and has not test cases addressing it :+1:

Reply:
Reverted :+1: